remme/shared/router.py

```diff
@@ -330,8 +348,8 @@ def fetch_peers(self):
             metadata=self._get_metadata(response)
         )
 
-    # def fetch_status(self):
-    #     response = self._handle_response(
+    # async def fetch_status(self):
```

Comment:
Why is that commented?

Reply:
This related to protobuf which does not exist in sawtooth core. So implementation we have, but it will not be workable without corresponding protos. Need to wait for an update or remove this.